### PR TITLE
Pic 442 get breach

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**
Added length and length units to the "order" field
Added sentencing court name - had to use a new endpoint in community API for this

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -4,6 +4,6 @@ ignore:
   SNYK-DEBIAN10-GNUTLS28-609778:
     - '*':
         reason: Ticket to change base image raised.
-        expires: 2020-10-13T00:00:00.000Z
+        expires: 2020-10-30T00:00:00.000Z
```
Comment (jhevans) [inline .snyk:7]: Ah that's one thing I forgot to check when I did the image update is our Snyk policy.  I think we can remove this now. I've got another PR open so leave it to me, we can merge this as is.

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.justice.probation.courtcaseservice.service.model.KeyValue;
+
+@ApiModel(description = "Court Appearances from Community API")
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java:14]: Won't be shown in Swagger so I think we can remove this?
```suggestion
```

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearancesResponse.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.List;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@ApiModel(description = "Court Appearances from Community API")
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearancesResponse.java:12]: Same as previous - not Swaggerable?

```suggestion
```

**Review by jhevans — COMMENTED** [review]
Couple of redundant annotations but otherwise good

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java (hunk 2 above)
Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java:14]: Oh yes !

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearancesResponse.java (hunk 3 above)
Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearancesResponse.java:12]: Done

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+@Builder
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```
[the remark this answers is not in the document]
Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java:19]: Done

File: src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearancesResponse.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+@Builder
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```
[the remark this answers is not in the document]
Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearancesResponse.java]: done

File: .snyk (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply .snyk]: OK

**Review by jhevans — APPROVED** [review]

**Pull request merged**